tests/drivers/memc/ram/src/main.c

```diff
@@ -48,13 +48,26 @@ BUF_DEF(sram2);
 BUF_DEF(psram);
 #endif
 
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(sdram), okay)
```

Comment:
Does this run on the hardware with twister currently? Looking at the testcase.yaml: https://github.com/zephyrproject-rtos/zephyr/blob/305ceba9204d9a7ee4bae0a6103ef4cbd9a1c54e/tests/drivers/memc/ram/testcase.yaml#L2

It looks like you'd need to add a new entry to get this to run on the EK_RA8D1 (I'd suggest doing it by the DT compatible, like other testcases have done with their filter)

Reply:
Sorry I miss this comment. Updated